javascript/README.md

```diff
@@ -382,3 +382,21 @@ if ("0" == false) {console.log('foo')} // this will console log! bad!
 // good
 if ("0" === false) {console.log('foo')}
 ```
+
+##Calling array methods on array-like objects
+
+Use functions on Array.prototype for array-like objects. (eg. 'arguments' is not an array, but 
+is array-like).
+
+###Prefer using methods from `[]` over `Array.prototype`
+
+Performance between the two styles is almost identical. Prefer the short, more succinct version.
+
+```javascript
+// Good
+[].slice.call(arguments); 
```

Comment:
Hm. We might want to change this to something like `[].slice.call(this, arguments...);`. The first argument to call is the value of this to bind the function to, not the arguments that are actually passed to `slice`.